data/net/minecraft/world/level/chunk/storage/EntityStorage.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/world/level/chunk/storage/EntityStorage
+	METHOD <init> (Lnet/minecraft/server/level/ServerLevel;Ljava/io/File;Lcom/mojang/datafixers/DataFixer;ZLjava/util/concurrent/Executor;)V
+		ARG 1 level
+		ARG 2 regionFolder
+		ARG 3 fixerUpper
+		ARG 4 sync
+		ARG 5 executor
```

Comment:
Something brought up in the mentioned PR is the (yet-to-be-fully-resolved) point of 'parameters follow field names': `executor` -> `mainThreadExecutor`.